.github/workflows/nix.yml

```diff
@@ -0,0 +1,24 @@
+name: "Build nix"
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - compatible
+      - berkeley
+      - releases/**
+      - main
```

Comment:
If I understand correctly, this action will be run after a merge to one of these branches, right? Shouldn't we rather aim to avoid breking Nix in the first place, by running the check before merge?

Reply:
I see. That's great. Thanks.